src/configs/index.js

```diff
@@ -1,46 +1,8 @@
+export const language = 1;
 export const itemsPerPage = [15, 30, 60];
```

Comment:
It is constant. Please, use the uppercased name

Reply:
deleted - it was unused